### PR TITLE
Add ankerpeet bio to the a.json file

### PR DESCRIPTION
**Pull request by ankerpeet**

### CODE REVIEW

File: mentors/javascript/a.json [hunk 1]
```diff
@@ -70,5 +70,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I use JavaScript (React) at work and did Java, Python, PHP in the past. I'm always excited to learn and help!"
+  },
+  {
+    "github_username": "ankerpeet",
+    "name": "Anker Peet",
+    "link_text": "My blog - techbox32",
+    "link_url": "www.techbox32.com",
```
Comment (NobbZ) [inline mentors/javascript/a.json]: You are missing a schema here.

Comment (ankerpeet) [reply mentors/javascript/a.json]: What am I missing?

Comment (rpottsoh) [reply mentors/javascript/a.json]: Either `http://` or `https://`

Comment (ankerpeet) [reply mentors/javascript/a.json]: I have added http:// to the URL

**Pull request merged**